### DISCUSSION TITLE
Request for Guidance on Recreating CVE-2023-0735 in Wallabag 2.5.3

### DISCUSSION THREAD

**Discussion by kevinpvo** [wallabag/wallabag#6433] [Q&A] [answered]
Hi @j0k3r,

I am currently working on a cybersecurity project (in a team of 4 graduate students at CU Boulder including myself) where we aim to recreate the bug found at https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-0735 for Wallabag version 2.5.3. We would appreciate any guidance on how to proceed and whether it is possible to do so.

Specifically, we would like to know how to install the version in which this bug has not yet been patched, and then how to recreate the bug. We have attached our project proposal for your reference: 
[CYBR_5300_Team_A_Project_Proposal.pdf](https://github.com/wallabag/wallabag/files/11226806/CYBR_5300_Team_A_Project_Proposal.pdf)

Thank you in advance for your assistance.

Best regards,
Kevin

**Comment by j0k3r** [comment, marked as answer]
Hello @kevinpvo,

1. You can download the affected version here: https://github.com/wallabag/wallabag/releases/download/2.5.3/wallabag-2.5.3.tar.gz or use Docker to install the 2.5.3 from the given tags: https://hub.docker.com/r/wallabag/wallabag/tags
2. Read the doc here to install it (in "On shared hosting"): https://doc.wallabag.org/en/admin/installation/installation.html#on-shared-hosting
3. And, once installed, you can follow steps defined here (in "Steps to reproduce"): https://huntr.dev/bounties/8bc78cb1-b10b-4152-842e-ceb999fc5508/

**Reply by kevinpvo** [reply]
Thank you so much!
